scripts/org.ooni.probe.cli.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>org.ooni.probe.daily-run</string>
+
+    <key>KeepAlive</key>
+    <false/>
+    <key>RunAtLoad</key>
+    <true/>
+
+    <key>Program</key>
+    <string>/Users/USERNAME/.ooniprobe/ooniprobe</string>
+    <key>ProgramArguments</key>
```

Comment:
Isn't this duplicated?

Reply:
Oops, yeah. For some reason it works even if it has a duplicated key. Will fix it though.